odyssey-r155188-1.23/Source/JavaScriptCore/CMakeLists.txt

```diff
@@ -61,7 +61,7 @@ set_source_files_properties(${JSC_SRC}
     PROPERTIES COMPILE_FLAGS "-fno-exceptions"
 )
 add_library(jsc STATIC ${JSC_SRC})
-add_dependencies(jsc wtf wtfurl)
+add_dependencies(jsc wtf)
```

Comment:
This is necessary because `wtfurl` doesn't exist and newer versions of cmake shows it as an error breaking the compilation